storage/aws/aws.go

```diff
@@ -112,7 +111,9 @@ type sequencer interface {
 	currentTree(ctx context.Context) (uint64, []byte, error)
 
 	// nextIndex returns the next available index in the log.
-	nextIndex(ctx context.Context) (uint64, error)
```

Comment:
I think `nextIndex` is still needed.

Reply:
You're right, that looks like an accidental deletion - fixed.